src/main/java/com/virtudoc/web/controller/UserAccountController.java

```diff
@@ -52,5 +52,10 @@ public String registerUser(@ModelAttribute NewUserDTO userDTO, Errors errors) {
         return "redirect:/login"; // TODO: change to "check your emails" page
     }
 
+    @PostMapping("/welcome")
```

Comment:
Is this supposed to be where users are directed once they click the button in the verification email? This will need to be changed to a GET route. See my other comment for more details.